src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -32,6 +32,10 @@
       "longName": "tenant-id",
       "shortName": ""
     },
+    "DefaultScope": {
+      "longName": "default-scope",
+      "shortName": "s"
```

Comment:
I would avoid setting a shortname here, especially a short one such as `s`. Given that this isn't a required argument that's used in all scenarios, for example, templates where auth is not configured, we don't want to reserve a shortname.